### PR TITLE
Update Bundler Contributor Guidelines README

### PR DESCRIPTION
**Pull request by rubymorillo**
Thanks so much for the contribution!
To make reviewing this PR a bit easier, please fill out answers to the following questions.

### What was the end-user problem that led to this PR?

The problem was...
...the contributor guide was is too long and too confusing. It was difficult to determine what information needed to be reviewed in what sequence. We wanted to make the guidelines easy to read and easy to follow.

### What was your diagnosis of the problem?

My diagnosis was...
...to condense the information 
### What is your fix for the problem, implemented in this PR?

My fix...
to condense the information presented and truncate the number of sections. See changes:

Before
After

### Why did you choose this fix out of the possible options?

I chose this fix because...
...it's a quick win!

### CODE REVIEW

File: doc/contributing/README.md [hunk 1]
```diff
@@ -1,38 +1,27 @@
...
 
-Welcome to Bundler! We are so happy that you're here. We know joining a new open source project can be daunting, so here's a quick overview of what you can expect from this documentation.
+Thank you for your interest in making Bundler better! We welcome contributions from everyone. Dozens of contributors like you have submitted feature improvements, fixed bugs, and written new documentation. [Join the Bundler Slack community](http://slack.bundler.io/) to connect with the Bundler core team and other contributors like you.
 
-*Something missing?* Send us a [pull request](../development/PULL_REQUESTS.md)!
+Before submitting a contribution, please take a moment to read through the following guidelines:
 
-**Recommended first steps**
+* [Bundler Code of Conduct](https://github.com/bundler/bundler/blob/master/CODE_OF_CONDUCT.md) (By participating in Bundler, you agree to abide by the terms laid out in the CoC.)
+* [Development Setup Guidelines](https://github.com/bundler/bundler/blob/master/doc/development/SETUP.md)
```
Comment (hmistry) [inline doc/contributing/README.md]: Development Setup Guidelines is more of instructions on how to get setup than guidelines. As such, it is the odd one in this list. I presume this list is a set of high level guidelines about conduct of conduct, guidelines on posting issues and PR's, not how to setup your machine to develop.

I'd suggest finding another place for this bullet.

File: doc/contributing/README.md [hunk 2]
```diff
@@ -1,38 +1,27 @@
...
-  - Checkout [How you can help: your first contributions!](HOW_YOU_CAN_HELP.md) for a list of suggestions to get started
-- *If you're interested in helping with documentation:*
-  - Read up on [how documentation works for Bundler](../documentation/README.md)
-  - Learn about our [documentation vision](../documentation/VISION.md)
 
-You can start learning about Bundler by reading [the documentation](http://bundler.io). If you want, you can also read a (lengthy) explanation of [why Bundler exists and what it does](http://bundler.io/rationale.html).
+## Contributing to Bundler
 
-## [How you can help: your first contributions!](HOW_YOU_CAN_HELP.md)
+Now that you’ve looked over our contribution guidelines, here are the different ways you can start contributing to the Bundler project:
```
Comment (hmistry) [inline doc/contributing/README.md]: I think you mean contributor guidelines. However, I'd leave out the first half of the sentence, "Now that.... guidelines" to keep it simple.

**Comment by rubymorillo** [comment]
@hmistry per your comments I've made the requested updates. Thank you!

**Comment by colby-swandale** [comment]
Thanks! @bundlerbot r+

**Pull request merged**